### PR TITLE
API doc for multi-org usage endpoint

### PR DESCRIPTION
**Pull request by carlmartensen**
Documentation for the Multi-Org Usage API. The PM on this wrote the content.

### CODE REVIEW

File: content/api/usage/usage_multi_org_code.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: Get Usage across your Multi-Org Account
+type: apicode
+order: 22.4
+external_redirect: /api/#get-usage-across-your-multi-org-account
+---
+
+##### Signature
+`GET https://app.datadoghq.com/api/v1/usage/summary`
+##### Example Request
+{{< code-snippets basename="api-billing-usage-summary" >}}
```
Comment (l0k0ms) [inline content/api/usage/usage_multi_org_code.md]: It seems that there is no code snippets file for this end-point. 
Could you add at least the sh example and add disclaimers for py and rb code snippets? many thanks 🙇

Comment (carlmartensen) [reply content/api/usage/usage_multi_org_code.md]: @l0k0ms Because its in private beta we haven't included code snippets for any of the usage endpoints

**Comment by carlmartensen** [comment]
@l0k0ms Made the requested changes. Ready for re-review. Thanks for reviewing.

**Comment by l0k0ms** [comment]
Preview: https://docs-staging.datadoghq.com/carl/multi-org-usage-api/api/?lang=bash#get-usage-across-your-multi-org-account

**Comment by carlmartensen** [comment]
@jeremy-lq What do you mean?

**Comment by AndrewMcBurney** [comment]
He means you should substitute `Usage` for `usage` globally. It's a vim regex.

File: content/api/usage/usage_multi_org.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Get Usage across your Multi-Org Account
+type: apicontent
+order: 22.4
+external_redirect: /api/#get-usage-across-your-multi-org-account
+---
+
+## Get Usage across your Multi-Org Account
+
+Get Usage across your Multi-Org Account
```
Comment (AndrewMcBurney) [inline content/api/usage/usage_multi_org.md]: Could you please make this a lowercase `usage`?

File: content/api/usage/usage_multi_org.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Get Usage across your Multi-Org Account
+type: apicontent
+order: 22.4
+external_redirect: /api/#get-usage-across-your-multi-org-account
+---
+
+## Get Usage across your Multi-Org Account
```
Comment (AndrewMcBurney) [inline content/api/usage/usage_multi_org.md]: Could you please make this a lowercase `usage`?

File: content/api/usage/usage_multi_org.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: Get Usage across your Multi-Org Account
```
Comment (AndrewMcBurney) [inline content/api/usage/usage_multi_org.md]: Could you please make this a lowercase `usage`?

**Pull request merged**